audit/projects/k8s-infra-prow-build/services/compute/project-info.json

```diff
@@ -6,12 +6,12 @@
         "value": "default:10.32.0.0/14"
       },
       {
-        "key": "sshKeys",
-        "value": "gke-8d7225b757872b0f06a6:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQD2x5DoWU6zKsjIs0srrMdc3CupZyCk6uiJUZ6KeRb3s7MmKQ3GCngTaTmRT3eJJwvzGBHbqAGHayQVIDX8NIw8PBh7qS1wtwZXGiRGkQM9xZGnDGlLDh5/OPgsAZxvmO9WX0y1voN5Q9k26J47aVC/cVQetMD0UVYYE+fACq6Nwz7KtBoFuVE99e9QaqvJc7g3WzeOFcRj8FPtFKSskwVOIgbR4sEdY/TAlibyM3eTnnjIAKugNX78R+V9rSXDOl/OCAkx22Cy+TrF5VVzvGR4r8MytyoYupr6DLnNIqtPPnzQbx+GA+x4NBGjOB/MGidY2lTIc+qAoXX1aSOrQ+ff gke-8d7225b757872b0f06a6@gke-8d7225b757872b0f06a6\ngke-8d7225b757872b0f06a6:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC0TwjNFdXL0OC2Qt0dfP64bpc6Yy5vH353AiZ/0V9WJJ5n7Qrk7oQbbtGXYuWYs/+cPx7sA3jGh6CRNkPPTl/0gVrud1XM07gMkBlr8ulyHUIBFG3jEmDipedNHYIioVz6w4ZJhQcDe83n+MTW8a2iNFU8ihqfmjMcyj5q4q8c1jA1zWv06cKG3rlfTHM7E1Y1g/twmgymNkj55vMHfax3Zdh3zsjBqAsAhWLTCCPLpU39qXg90uDnzunoXQ0rmGVliJyhrHfeJlY5trC8C1M0dI4DVRpeT2PtS9YTMoJ017hA+To848af9S5bnpQ0eHhqaMGsAjwuaxULXIPEZZqz gke-8d7225b757872b0f06a6@gke-8d7225b757872b0f06a6\ngke-8d7225b757872b0f06a6:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDbYis3+c9yPGIVnC1PzVTlLVFf+8C67ElBXCFm75E6H3J3chHxAk8kc9VKEk6MoqNEL1xWH9B0GI+IeWcqJC8IhaPAv/FDqub2IUf0YZCUnjU48Y709DodM1+IQZZcmM4bDaiRzGoB+0H9TI2rL2jhb17xckvztWLTf4aKxgRUWn655FORnrcs7064kH8c4c+W+qgUlG1P5Vad6H1EmnFG6AB488kf9y5dE7A1tDQqX/Ssf5atcGlHMR/5gdTqEF2nx40j1btDBAlHOBSBOEsNOHe7xGKi+KDlyvJEjuZbq7yVY36+twXalxwO+b2wRjV++OZ1fk9oKE9CJjxMA+1r gke-8d7225b757872b0f06a6@gke-8d7225b757872b0f06a6"
+        "key": "ssh-keys",
+        "value": "ameukam:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDW9HdP9R3Ro8WFSFVUan1NFZV7vDp/LglFpydX1zv4sErEnV+iS3TPlRId/3GXsxTmkWRUG4e1zFarnn9UAaUFXy1woWQqeZ8YPWqBQYQ6S7FRhIxWhWDqzyGdGOxKaiD/i6S+sllVUzIEuFRYTzZf54V8r+sr3M4X0zWCF2EjCpE2D7YJcpXuo/QUJiTgdk0PbmeShainuXsP0pOX6tvpSdOVJN+ZPu9b5dBBlW0B9WpdN2qt7qx9lfaqUa6kIlTuq9ggPGGifcVBK5A9S+VbzbWBetz6simidz9ED5cA+H9Mlo4DTgLeSFNLeOpSoTEuVRnIyVsg9EPLrhARrDTpDlFFT/ZIROGghbyK3REpX0nD4NCMSkdroG/RsrcTsRv/ro4PFpEJBzanlWNat4ymtdPOCvlrPjiMvWg63ENxti+aPgyp5aVs20aYhQEnE/+rWAGDm+CnRIEyQK6gQ4k7p+KRxOc3MG8dKUZ+tlZWmdBlAA5Gf/qxVwXBWSQbLXc= ameukam@cs-1059757242650-default"
```

Comment:
Did some analysis related to https://github.com/kubernetes/kubernetes/issues/109038. See  https://kubernetes.slack.com/archives/CCK68P2Q2/p1648798865393099